src/content/learn/installation.md

```diff
@@ -1,25 +1,25 @@
 ---
-title: Installation
+title: Instalace
 ---
 
 <Intro>
 
-React has been designed from the start for gradual adoption. You can use as little or as much React as you need. Whether you want to get a taste of React, add some interactivity to an HTML page, or start a complex React-powered app, this section will help you get started.
+React byl od začátku navržen pro postupnou integraci. Můžete používat tolik Reactu, kolik potřebujete. Ať už chcete získat představu o Reactu, přidat nějakou interaktivitu do HTML stránky, nebo začít komplexní aplikaci poháněnou Reactem, tato sekce vám pomůže začít.
```

Comment:
I am not sure about the sentence "Můžete používat tolik Reactu, kolik potřebujete." Maybe "Můžete využívat z Reactu tolik, kolik potřebujete."?